src/pointgroup.c

```diff
@@ -401,8 +401,7 @@ Pointgroup ptg_get_transformation_matrix(int transform_mat[3][3],
         get_axes(axes, pointgroup.laue, &pointsym, aperiodic_axis);
         set_transformation_matrix(transform_mat, axes);
     } else {
-        warning_print("spglib: No point group was found ");
-        warning_print("(line %d, %s).\n", __LINE__, __FILE__);
+        warning_print("spglib: No point group was found\n");
```

Comment:
This one is a good example of the things that need to be reviewed:

From https://github.com/spglib/spglib/issues/338#issuecomment-2023414189, it seems that this message is one of those that is not related to an error. Thus in a normal run, this message gets spammed a lot. Do you think it should be changed to a debug message?

Reply:
Fwiw, in my opinion this message should be changed to a debug message, that ideally is only printed if the overall routine fails or the code is being run in some debug type mode. Thanks for looking into this @LecrisUT!